pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -330,8 +330,7 @@ type Volumes interface {
 	// Check if the volume is already attached to the node with the specified NodeName
 	DiskIsAttached(diskName KubernetesVolumeID, nodeName types.NodeName) (bool, error)
 
-	// Check if a list of volumes are attached to the node with the specified NodeName
-	DisksAreAttached(diskNames []KubernetesVolumeID, nodeName types.NodeName) (map[KubernetesVolumeID]bool, error)
+	CheckClusterDisks(map[types.NodeName][]KubernetesVolumeID) (map[types.NodeName]map[KubernetesVolumeID]bool, error)
```

Comment:
`DisksAreAttached` is internal implementation detail, it just happens to be used in other cloud providers too because people copied some of the ideas of implementation. I don't think `DisksAreAttached` is part of `VolumePlugin` interface or anything.